blocks/components/editable/index.js

```diff
@@ -10,6 +10,7 @@ import { Parser as HtmlToReactParser } from 'html-to-react';
  */
 import './style.scss';
 
+const KEYCODE_BACKSPACE = 8;
```

Comment:
There's also `window.tinymce.util.VK`. It's okay like this I guess, but at some point we may have to use other things from it anyway, like `metaKeyPressed`.